modules/apps/site-initializer/site-initializer-liferay-learn/src/main/resources/site-initializer/ddm-templates/navigation-utilities/ddm-template.ftl

```diff
@@ -64,14 +64,10 @@
 											<div class="adt-submenu-item-preheader color-neutral-3 font-weight-semi-bold">${preheaderText}</div>
 										</#if>
 
-										<#if stringUtil.equals(menuItemType, "Image")>
```

Comment:
Are we removing the ability to add images to the navigation?

Reply:
For this section that was removed, it was for the heading change if the menu item type was an image
`										<#if stringUtil.equals(menuItemType, "Image")>
											<div class="adt-submenu-item-title font-size-paragraph-small font-weight-semi-bold">${navTertiaryItem.getName()}</div>
										<#else>
											<div class="adt-submenu-item-title font-size-paragraph-base font-weight-semi-bold">${navTertiaryItem.getName()}</div>
										</#if>`

Reply:
But this doesn't apply to the new navigation